src/workloads/query/TelemetryWithRedaction.yml

```diff
@@ -0,0 +1,158 @@
+  Config:
+    Name: {^Parameter: {Name: "Name", Default: "TelemetryAggregation"}}
+    Type: AdminCommand
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: 1024}}]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          Repeat: 1000
+          Database: admin
+          Operations:
+          - OperationMetricsName: TelemetryWithRedactionMetrics
+            OperationName: AdminCommand
+            OperationCommand:
+              aggregate: 1
+              pipeline: [
+                      {
+                        $telemetry: {
+                          redactIdentifiers: {^Parameter: {Name: "redactIdentifiers", Default: false}},
+                          redactionKey: *binDataGen
+                        }
+                      }
+                    ]
+              cursor: {}
+
+Actors:
+- Name: ClearCollection
+  Type: CrudActor
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Threads: *Threads
+        Collection: *Collection
+        Operations:
+        - OperationName: drop
+
+- Name: InsertData
+  Type: Loader
+  Threads: 4
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        MultipleThreadsPerCollection: true
+        CollectionCount: 1
+        DocumentCount: *DocumentCount
+        BatchSize: 1000
+        Document:
+          a: &integer {^RandomInt: {min: -100, max: 100}}
+          b: *integer
+          string: &string {^RandomString: {length: 5}}
+        Indexes:
+          - keys: {a: 1, b: 1}
+          - keys: {string: 1}
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+
+# Run some commands so that telemetry is collected.
+- Name: AggregationCommand
+  Type: CrudActor
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [3]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1000
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: PipelineWithoutOptions
+          OperationName: aggregate
+          OperationCommand:
+            Pipeline: [{$match: {a: 42}}, {$group: {_id: "$b", sum: {$sum: "$a"}}}]
+
+- Name: FindCommand
+  Type: CrudActor
+  Threads: 1
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [4]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1000
+        Collection: *Collection
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            Filter: {string: {$in: ["hello", "hi", "bye"]}}
+
+# Telemetry with redaction.
```

Comment:
Oh so it looks like here you're running the telemetry actor in its own phase?

I think the idea of the ticket was to see if we make everything else slower if we run it concurrently. So that would be something like
Phase 3: Run a bunch of concurrent finds on many threads - just maxing out load mimicing a stressed out cluster
Phase 4: Quiesce
Phase 5: Run all the same actors with all the same numbers of threads, but now add an actor running $telemetry also. How much slower does it get?

What might be interesting and hard about this though - should we repeat telemetry a lot in a loop? That's not really how we expect it to be used. We can start with a telemetry actor just repeating itself for the duration of the phase (you probably want to make it a duration in minutes rather than a "repeat" number? so they all run for the same amount of time?). It's worth checking if there's any way to get an actor to repeat N times but with some sort of sleep/delay between invokations? Like every 5 seconds say for a 1 minute trial workload?